packages/flutter/test/widgets/sliver_coordinator_test.dart

```diff
@@ -0,0 +1,196 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+
+void main() {
+  testWidgets('SliverCoordinator basics', (WidgetTester tester) async {
```

Comment:
Same as the others, this will need every permutation of vertical/horizontal - reversed/not.

Reply:
Done.